perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -20,8 +20,18 @@
 import logging
 from perfkitbenchmarker import configs
 from perfkitbenchmarker import sample
+from perfkitbenchmarker import flags
+
 import re
 
+flags.DEFINE_enum('ping_net_interface_type', 'internal_ip',
```

Comment:
Could you re-use the `--ip_addresses` flag defined in [`vm_util`](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/vm_util.py#L95-L100)? We already have a lot of flags :)